ui/src/button.tsx

```diff
@@ -125,6 +135,20 @@ const
           wave.push()
         }
       },
+      onClick = handleOnClick(name, value, path),
+      getItemProps: (commands: Command[]) => Fluent.IContextualMenuItem[] = commands =>
+        commands.map(({ name, label, caption, icon, value, data, items }) => ({
+          key: name,
+          text: label,
+          title: caption,
+          iconProps: { iconName: icon },
+          data: value ?? data, disabled,
```

Comment:
is this `data` used anywhere?

Reply:
Nope, we are using `value` inside onClick handler. Removing.